### PR TITLE
Update metadata based on debriefing

### PR DESCRIPTION
**Pull request by CunliangGeng**
- Updated the description, especially the expectation part
- Updated the schedule and syllabus based on the time tracking

### CODE REVIEW

**Review by lyashevska — APPROVED** [review]
looks good

File: ds-sklearn/description.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 so that upon completion, you will be able to train your first machine learning models and understand what next steps to take to improve them.
 
-We start with processing the data so that it is suitable for machine learning.
-Then we learn how to fit a model to the data using scikit-learn.
-We learn how to select the best model, learn about different machine learning models,
+We start with data exploration so that it is suitable for machine learning.
+Then we learn how to train a model on the data using scikit-learn.
+We learn how to select the best model from the trained models, learn how to
+use different machine learning models (like linear regression, logistic regression,
+decision tree and support vector machine),
```
Comment (DaniBodor) [inline ds-sklearn/description.md:13]:
```suggestion
We learn how to select the best model from the trained models and how to
use different machine learning models (like linear regression, logistic regression,
decision tree and support vector machine),
```

File: ds-sklearn/description.md [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 and discuss some of the best practices when starting your own machine learning project.
+
+The workshop will NOT cover the following topics:
+- math and theory on machine learning
+- data proprocessing, e.g. data cleaning, handling missing values
+- deep learning models
```
Comment (DaniBodor) [inline ds-sklearn/description.md:19]: Very good addition!
Should we add references to external courses that cover that? Or is that not our responsibility?

**Review by DaniBodor — APPROVED** [review]
Looks good! 👍 

(sorry for being slow with this)

File: ds-sklearn/description.md (hunk 2 above)
Comment (CunliangGeng) [reply ds-sklearn/description.md:19]: No, we don't add the external courses here (it's our advertisement).
We added them to the [resources.md](https://github.com/esciencecenter-digital-skills/workshop-metadata/blob/main/ds-sklearn/resources.md).  We also send out the further study resources to participants after the workshop.

**Pull request merged**